src/views/LoginSuccess.vue

```diff
@@ -90,20 +94,31 @@ export default class LoginSuccess extends Vue {
             })),
         };
 
-        if (this.gotError) {
+        if (this.receiptsError) {
+            this.title = 'Your addresses may be\nincomplete.';
             this.state = Loader.State.WARNING;
-            this.title = `Your addresses may be
-incomplete.`;
         } else {
             this.title = 'Your account is ready.';
             this.state = Loader.State.SUCCESS;
+            setTimeout(this.resolveResult.bind(this), Loader.SUCCESS_REDIRECT_DELAY);
         }
-
-        setTimeout(() => this.$rpc.resolve(result), Loader.SUCCESS_REDIRECT_DELAY);
     }
 
     private get status() {
         return !this.retrievalFailed ? 'Connecting to Nimiq...' : 'Address retrieval failed. Retrying...';
     }
+
+    private get message() {
+        return this.receiptsError && 'We might have missed used addresses that have no balance.';
+    }
+
+    private get action() {
+        return this.receiptsError && 'Check account';
```

Comment:
I'd call this action `Go to account` I think, because with 'Check account' I would expect that the app does another check.